vehicle/mkv/software/throttle/throttle.c

```diff
@@ -3,31 +3,39 @@
 #include "libs/timer/api.h"
 #include <avr/interrupt.h>
 
-#include "throttle_config.h"
 #include "libs/gpio/api.h"
+#include "throttle_config.h"
 
 #include <stdbool.h>
 #include <stdint.h>
 
+// Define throttle values to use when mapping
+// Values are capped at this range
+// Recheck/set these values
+#define throttle1_min_voltage (218)
+#define throttle1_max_voltage (654)
+#define throttle2_min_voltage (298)
+#define throttle2_max_voltage (1016)
+
+uint8_t CANMotorController[8] = { 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
Change to

```c
uint8_t can_motor_controller[8] = { 0 };
```

Does the same thing and adheres to our rule of using snake_case.

Reply:
Also this should be defined in `throttle_config.h`